code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -68,6 +68,63 @@
 /obj/structure/bed/post_unbuckle_mob(mob/living/M)
 	M.pixel_y = M.get_standard_pixel_y_offset()
 
+/obj/structure/bed/shove_impact(mob/living/target, mob/living/attacker)
+	. = ..()
+	if(!ishuman(target))
+		return
```

Comment:
this needs to check if the person is already sleeping, or else you can just spam this I'm pretty sure

Reply:
Addressed in https://github.com/ParadiseSS13/Paradise/pull/20436/commits/429c4b65b3fe99bc82a47401230224972c1597b6, you can still shove folks to bed while they're asleep but they won't get any sleepier if you do